.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,52 @@
+name: Feature Request
+description: Suggest an idea for substra
+title: "ENH: "
+labels: [Enhancement]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Feature Type
+      description: Please check what type of feature request you would like to propose.
+      options:
+        - label: >
+            Adding new functionality to substra
+        - label: >
+            Changing existing functionality in substra
+        - label: >
+            Removing existing functionality in substra
+  - type: textarea
+    id: description
+    attributes:
+      label: Problem Description
+      description: >
+        Please describe what problem the feature would solve, e.g. "I wish I could use substra to ..."
+      placeholder: >
+        A clear and concise description of what the problem is.
+    validations:
+      required: true
+  - type: textarea
+    id: feature
+    attributes:
+      label: Feature Description
+      description: >
+        Please describe how the new feature would be implemented, using pseudocode if relevant.
```

Comment:
I think feature description is really what the feature will do: for instance having a new page in the frontend that shows the whole network, and so is different to "Please describe how the new feature would be implemented, using pseudocode if relevant."

Reply:
```suggestion
      label: Feature Description
      description: >
        Please describe what will be this new feature.
```